src/typescript/frontend/src/components/pages/emojicoin/components/chat/components/message-container/index.tsx

```diff
@@ -36,7 +36,7 @@ const MessageContainer: React.FC<MessageContainerProps> = ({
     }
     const selfName = nameResolver(account.address);
     setFromAnotherUser(selfName !== message.sender);
-  }, [account, fromAnotherUser, message, nameResolver]);
+  }, [account, message]);
```

Comment:
It triggered infinite rerenders. I could alternatively put just nameResolver instead of account. I think this would be better since nameResolver loads after account and this way when it renders it would have fresh values for both.

Reply:
Actually no, just putting nameResolver causes infinite rendering